.gitignore

```diff
@@ -4,3 +4,6 @@ tmp
 docs/build
 .ruby-version
 node_modules
+
+.idea
+ember-simple-auth.iml
```

Comment:
I changed my PR to have a newline at the end, for some reason git or github is removing it or not displaying it.